File: KitDemo/KitDemo/WelcomeViewController.m
//
//  WeicomeViewController.m
//  KitDemo
//
//  Created by Hu on 16/8/2.
//  Copyright © 2016年 redye. All rights reserved.
//

#import "WelcomeViewController.h"
#import "YHWelcomeView.h"

@interface WelcomeViewController ()
@end

@implementation WelcomeViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)setUI {
    [super setUI];
    self.navigationController.navigationBar.hidden = YES;

    YHWelcomeView *welcomeView = [[YHWelcomeView alloc] initWithPrefix:@"ind_upgrade" count:5 complete:nil];
    [welcomeView showInView:self.view];
}

- (BOOL)prefersStatusBarHidden {
    return YES;
}


@end

File: KitDemo-Swift/KitDemo-Swift/WelcomeViewController.swift
//
//  WelcomeViewController.swift
//  KitDemo-Swift
//
//  Created by Hu on 16/8/3.
//  Copyright © 2016年 redye. All rights reserved.
//

import UIKit

class WelcomeViewController: BaseViewController {

    override func setUI() {
        super.setUI()
        self.navigationController?.isNavigationBarHidden = true
        
        let welcomeView = YHWelcomeView(frame: self.view.bounds, prefix: "ind_upgrade", count: 5)
        welcomeView.complete = {
            print("完成")
        }
        welcomeView.showInView(self.view)
    }
    
    override var prefersStatusBarHidden : Bool {
        return true
    }
}
